### ISSUE TITLE
Config file and certs can be pushed to, but not pulled from MongoDB

### ISSUE THREAD

**Issue by whalehub** [Ylianst/MeshCentral#106]
I was browsing the MeshCentral user guide today when I discovered that you can store the config file and certs in the MongoDB database to encrypt them at rest. I pushed the config file and certs to the database with the following command:

```
node /opt/meshcentral/node_modules/meshcentral --dbpushconfigfiles /opt/meshcentral/meshcentral-data --configkey <redacted>
```

The import of the files worked fine. Afterwards, I ran MeshCentral with the following command to pull them from the database at runtime:

```
node /opt/meshcentral/node_modules/meshcentral --loadconfigfromdb <redacted> --mongodb "mongodb://127.0.0.1:27017/meshcentral"
```

MeshCentral was able to successfully pull the files from the database this way. So far so good. Next, I wanted to make an edit to the config file and to pull it from the MongoDB database in order to do that. To achieves this, I ran the following command, but instead of the config file being saved to the specified folder, I simply received the error message "File not found."

```
node /opt/meshcentral/node_modules/meshcentral --dbpullconfigfiles /opt/meshcentral/meshcentral-data --configkey <redacted>
```

I tried it again, this time also specifying that I want to pull the files from MongoDB instead of NeDB, but I received the same error message.

```
node /opt/meshcentral/node_modules/meshcentral --dbpullconfigfiles /opt/meshcentral/meshcentral-data --configkey <redacted> --mongodb "mongodb://127.0.0.1:27017/meshcentral"
```

Is this a bug or am I doing something wrong?

**Comment by Ylianst** [comment]
Published MeshCentral v0.2.9-e that includes a fix for this. The old code will really bad, this should improve it a bunch. Please let me know if you tested it and it worked.

**Comment by whalehub** [comment]
@Ylianst I can confirm that it works now.
